### PR TITLE
Update Email and add Files Domain

### PR DESCRIPTION
**Pull request by ghost**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

Screenshot 2023-04-23 23 44 28
Screenshot 2023-04-23 23 45 21

### CODE REVIEW

File: domains/files.liquid.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "ThinLiquid",
+        "email": "thinliquid@proton.me"
+    },
+    "record": {
+        "CNAME": "dns.tixte.com",
+        "TXT": "_tixte-challenge=6ZzT79WA"
```
Comment (WilliamDavidHarrison) [inline domains/files.liquid.json:8]: We don't support TXT verification.

File: domains/liquid.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "owner": {
         "username": "ThinLiquid",
-        "email": "ayomideilesanmi10@gmail.com"
+        "email": "thinliquid@proton.me"
     },
     "record": {
-        "CNAME": "glitch.edgeapp.net"
+        "CNAME": "9b0994e0-3427-49c7-8609-b7027badd871.id.repl.co"
```
Comment (WilliamDavidHarrison) [inline domains/liquid.json:7]: I can't reach this website.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/files.liquid.json (hunk 1 above)
Comment (ThinLiquid) [reply domains/files.liquid.json:8]: oh

File: domains/liquid.json (hunk 2 above)
Comment (ThinLiquid) [reply domains/liquid.json:7]: it takes a while to load cause its replit

**Pull request closed without merging**